static/src/javascripts/projects/common/modules/email/run-checks.js

```diff
@@ -97,7 +98,8 @@ define([
             return page.keywordExists(['Football']) && allowedArticleStructure();
         },
         usBriefing: function () {
-            return config.page.section === 'us-news' && allowedArticleStructure();
+            return (config.page.section === 'us-news' || config.page.series === 'Guardian US briefing') &&
```

Comment:
Where does `config.page.series` come from? This `'Guardian US briefing'` seems a bit arbitrary, but maybe it's not such a big deal if `config.page.series` is coming from something static, like a tag.